irohad/model/converters/impl/json_query_factory.cpp

```diff
@@ -113,6 +116,25 @@ namespace iroha {
             | toQuery;
       }
 
+      optional_ptr<Query> JsonQueryFactory::deserializeGetTransactions(
+          const Value &obj_query) {
+        auto des = makeFieldDeserializer(obj_query);
+        return make_optional_ptr<GetTransactions>()
+            | des.Array(
+                  &GetTransactions::tx_hashes,
+                  "tx_hashes",
+                  [](auto tx_hashes) {
+                    return deserializeArray<std::vector<iroha::hash256_t>>(
```

Comment:
Not really clear for me: why `des.Array` isn't used?

Reply:
Thanks.
`iroha::hash256_t` is expressed as hash but the feature of ignoring invalid hash or discarding query is API dependent. If we decide invalid hash will be skipped in `Convert<std::vector<iroha::hash256_t>>`, another API should follow this feature.

But, currently, `std::vector<iroha::hash256_t>` is used in only `GetTransactions` model. So I can change the code.